### ISSUE TITLE
redundantParens removes required parentheses for function call after #selector

### ISSUE THREAD

**Issue by ChristopherRogers** [nicklockwood/SwiftFormat#226]
SwiftFormat will remove the parentheses on the function call `foo()` below:

```swift
func foo() -> Bool { return true }

if action == #selector(UIResponderStandardEditActions.paste(_:)) && foo() {
}
```

**Comment by nicklockwood** [comment]
@ChristopherRogers thanks for reporting it, I'll get this fixed asap.

**Comment by nicklockwood** [comment]
Fixed in 0.33.3

**Comment by ChristopherRogers** [comment]
That was fast! Thank you!
